tests/aws/services/lambda_/event_source_mapping/test_cfn_resource.validation.json

```
@@ -1,5 +1,11 @@
 {
   "tests/aws/services/lambda_/event_source_mapping/test_cfn_resource.py::test_adding_tags": {
-    "last_validated_date": "2024-11-06T11:55:29+00:00"
+    "last_validated_date": "2025-05-19T08:31:57+00:00",
+    "durations_by_phase": {
```

```suggestion
    "durations_in_sec": {
```
Just a suggestion. Having the units makes it easier to read IMO - and the fact that this is by phase is already obvious from the content of the dict.

Love the initiative btw! Very useful to include this data in the test suite.